routing/routing_benchmarks/helpers.cpp

```diff
@@ -0,0 +1,143 @@
+#include "routing/routing_benchmarks/helpers.hpp"
+
+#include "testing/testing.hpp"
+
+#include "routing/base/astar_algorithm.hpp"
+#include "routing/features_road_graph.hpp"
+#include "routing/route.hpp"
+#include "routing/router_delegate.hpp"
+
+#include "indexer/classificator_loader.hpp"
+#include "indexer/mwm_set.hpp"
+
+#include "platform/local_country_file.hpp"
+#include "platform/local_country_file_utils.hpp"
+
+#include "geometry/polyline2d.hpp"
+
+#include "base/logging.hpp"
+#include "base/math.hpp"
+#include "base/timer.hpp"
+
+#include "std/limits.hpp"
+
+namespace
+{
+void TestRouter(routing::IRouter & router, m2::PointD const & startPos, m2::PointD const & finalPos,
+                routing::Route & foundRoute)
```

Comment:
No, I don't want to change signatures here. As you can see this pull request fixes problem with fake edges and adds a single (!) bicycle test. The whole routing tests refactoring is not the duty of this pull request.